pf4j/src/main/java/org/pf4j/AbstractPluginManager.java

```diff
@@ -238,6 +238,7 @@ public void loadPlugins() {
     /**
      * Unload all plugins
      */
+    @Override
```

Comment:
No need to override the default implementation of this method. See my next comment.